app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -86,8 +86,9 @@ class BookmarksToolbar extends ImmutableComponent {
       }
     }
     if (e.dataTransfer.files.length > 0) {
```

Comment:
Here we should probably change from `e.dataTransfer.files.length` into `e.dataTransfer.items.length`